mobile/library/cc/engine_builder.cc

```diff
@@ -328,6 +328,11 @@ EngineBuilder& EngineBuilder::enableInterfaceBinding(bool interface_binding_on)
   return *this;
 }
 
+EngineBuilder& EngineBuilder::setUseGroIfAvailable(bool value) {
```

Comment:
nit: For consistency with the other functions, maybe call this `user_gro_if_available` instead of `value`?

Reply:
done